src/store/map/index.js

```diff
@@ -57,50 +57,54 @@ const getters = {
 };
 
 const mutations = {
-    'update-location'(state, data) {
-        const newData = [...state.recentSearch, ...data];
-        state.locations = [...newData];
+    'update-location'(state, location) {
+        const localLocation = [...state.recentSearch, ...location];
+        state.location = [...localLocation];
     },
 
-    'update-selected-location'(state, data) {
-        state.selectedLocation.locName = data.place_name;
+    'update-selected-location'(state, location) {
+        state.selectedLocation.locName = location.place_name;
 
-        const lsRecentID = JSON.parse(localStorage.getItem('recentID'));
-        if (lsRecentID !== undefined) {
-            state.recentID = lsRecentID;
-        }
-
-        const objData = {
-            id: state.recentID,
+        const newLocalRecentSearch = {
```

Comment:
does local in the name mean variable local in the scope of method.
If so, then its misleading

Reply:
yes the variable has the local scope. How so it is misleading

Reply:
done

Reply:
please consider using _ instead local as its short and has no other meaning on contrary local has its own meaning too.
https://www.robinwieruch.de/javascript-naming-conventions/